test/e2e/scheduling/priorities.go

```diff
@@ -103,10 +106,20 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 
 		ginkgo.By("Trying to apply a label on the found node.")
 		k := fmt.Sprintf("kubernetes.io/e2e-%s", "node-topologyKey")
-		v := "topologyvalue"
+		v := "topologyvalue1"
 		framework.AddOrUpdateLabelOnNode(cs, nodeName, k, v)
 		framework.ExpectNodeHasLabel(cs, nodeName, k, v)
 		defer framework.RemoveLabelOffNode(cs, nodeName, k)
+
+		ginkgo.By("Trying to apply a label on the other node.")
+		v = "topologyvalue2"
```

Comment:
Why do we need to put different value(`topologyvalue2`) here?
According to https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity anti-affinity feature considers the key name only for specifying it as `TopologyKey` in the pod spec.
(I mean `The pod anti-affinity rule says that the pod prefers not to be scheduled onto a node if that node is already running a pod with label having key “security” and value “S2”. (If the topologyKey were failure-domain.beta.kubernetes.io/zone then it would mean that the pod cannot be scheduled onto a node if that node is in the same zone as a pod with label having key “security” and value “S2”.)` here)

Reply:
Thank you for your comment.
My understanding is as follows.
Nodes that the key specified in TopologyKey have same value are considered the same domain.
Pod anti-affinity feature schedules pods in a different domain if the pod is already running in the domain.
We need to put different value​(e.g. topologyvalue1 and topologyvalue2) to create two domains.

Reply:
Thanks for your explanation.
You are right. The document says it clearly according to `the pod cannot be scheduled onto a node if that node is in the same zone as a pod with label having key “security” and value “S2”.`
So the test needs to put different label to other nodes for making them different zone.